_layouts/frontpage.html

```diff
@@ -125,6 +109,7 @@ <h2><span>Scala in a Nutshell</span></h2>
         <div class="scala-item">
           <h3>{{scalaItem.shortTitle}}</h3>
           <p>{{scalaItem.shortDescription}}</p>
+          <div class="button">click to expand</div>
```

Comment:
I find “Click to expand” a bit long and repetitive. Do you think we could replace it with “…” for instance? (not sure “…” is self-explanatory enough, though)

Reply:
I think "More..." would be sufficient

Reply:
I have updated the PR description with a screenshot of using "More..."